src/resources/auth/auth.validator.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Validate sign up request
+ *
+ * @param req
+ * @param res
+ * @param next
+ */
+export const validateSignUp = (req, res, next) => {
+	req
+		.checkBody('email', 'Invalidi email')
+		.isEmail()
```

Comment:
Wondering if this is using legacy API:
https://express-validator.github.io/docs/legacy-api.html